packages/admin/admin/src/router/ConfirmationDialog.styles.ts

```diff
@@ -0,0 +1,39 @@
+import { createStyles } from "@mui/styles";
+
+import { RouterConfirmationDialogProps } from "./ConfirmationDialog";
+
+export type RouterConfirmationDialogClassKey =
+    | "root"
+    | "closeButton"
+    | "messageWrapper"
+    | "messageWarningIcon"
+    | "messageText"
+    | "actionButton"
+    | "saveButton"
+    | "discardButton";
+
+export const styles = () => {
+    return createStyles<RouterConfirmationDialogClassKey, RouterConfirmationDialogProps>({
+        root: {
+            "& .MuiDialog-paper": {
+                maxWidth: "25vw",
+            },
+        },
+        closeButton: {
+            position: "absolute",
+            right: 14,
+            top: 14,
+            color: "#fff",
+        },
+        messageWrapper: {
+            display: "flex",
+        },
+        messageWarningIcon: { fontSize: 20 },
+        messageText: { paddingLeft: 10 },
+        actionButton: {
+            minWidth: "47%",
+        },
+        saveButton: {},
+        discardButton: {},
```

Comment:
You could probably use a more styleguide-conform spacing by making each button 50% width with a margin, using `spacing()`  

sort of like this:

```suggestion
        actionButton: {
            flexGrow: 1,
            flexBasis: 50%,
        },
        saveButton: {
            marginLeft: theme.spacing(2),
        },
        discardButton: {
            marginRight: theme.spacing(2),
        },
```

or:

```suggestion
        actionButton: {
            width: `calc(50% - ${spacing(2)})`
        },
        saveButton: {
            alignSelf: "flex-end"
        },
        discardButton: {},
```

Reply:
Yes, that's better 👍